### PR TITLE
Extends wrapi/wrapf docs

### PR DESCRIPTION
**Pull request by Chaosus**
Added a bit more to descriptions and added notes about using modulo and fmod to achive same effect with better performance.

### CODE REVIEW

File: modules/gdscript/doc_classes/@GDScript.xml [hunk 1]
```diff
@@ -1241,6 +1241,16 @@
 				# infinite loop between 0.0 and 0.99
 				f = wrapf(f + 0.1, 0.0, 1.0)
 				[/codeblock]
+				[codeblock]
+				# infinite rotation (in radians)
+				angle = wrapf(angle + 0.1, 0.0, TAU)
+				[/codeblock]
+				Note: If you want to have a simple wrap behavior from 0.0 to 'n' (where 'n' is a positive float number) then its better for performance to use [method fmod] method like [code]fmod(number, n)[/code].
```
Comment (akien-mga) [inline modules/gdscript/doc_classes/@GDScript.xml]: Should be "it**'s** better".

File: modules/gdscript/doc_classes/@GDScript.xml [hunk 2]
```diff
@@ -1241,6 +1241,16 @@
...
+				The usage of this function is more flexible than using [method fmod] approach by giving the user a simple control over minimum value. Its also fully support negative numbers eg.
```
Comment (akien-mga) [inline modules/gdscript/doc_classes/@GDScript.xml]: Should be:

"The usage of <b>[code]wrapf[/code]</b> is more flexible than using **the** [method fmod] approach by giving the user ~~a simple~~ control over **the** minimum value. **It** also fully support**s** negative numbers<b>,</b> **e.g.**".

File: modules/gdscript/doc_classes/@GDScript.xml (hunk 1 above)
Comment (Chaosus) [reply modules/gdscript/doc_classes/@GDScript.xml]: Fixed

File: modules/gdscript/doc_classes/@GDScript.xml (hunk 2 above)
Comment (Chaosus) [reply modules/gdscript/doc_classes/@GDScript.xml]: Fixed

**Review by akien-mga — APPROVED** [review]

File: modules/gdscript/doc_classes/@GDScript.xml [hunk 3]
```diff
@@ -1241,6 +1241,16 @@
 				# infinite loop between 0.0 and 0.99
 				f = wrapf(f + 0.1, 0.0, 1.0)
 				[/codeblock]
+				[codeblock]
+				# infinite rotation (in radians)
+				angle = wrapf(angle + 0.1, 0.0, TAU)
+				[/codeblock]
+				Note: If you want to have a simple wrap behavior from 0.0 to 'n' (where 'n' is a positive float number) then it's better for performance to use [method fmod] method like [code]fmod(number, n)[/code].
```
Comment (homer666) [inline modules/gdscript/doc_classes/@GDScript.xml]: Might be better if it were phrased like this:

"Note: If you **just** want to ~have a simple~ wrap **between** 0.0 **and** [code]n[/code] (where [code]n[/code] is a positive float **value**)[...]"

Same goes for the other instance.

Comment (Chaosus) [reply modules/gdscript/doc_classes/@GDScript.xml]: @akien-mga WDYT ? (I'm not an english speaker, so I dont know)

Comment (akien-mga) [reply modules/gdscript/doc_classes/@GDScript.xml]: The suggested edit seems good yes.

Comment (Chaosus) [reply modules/gdscript/doc_classes/@GDScript.xml]: Fixed

**Review by StraToN — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.1.2.

**Pull request merged**